config/jobs/kubernetes-sigs/kubetest2/kubetest2-canaries.yaml

```diff
@@ -36,6 +36,43 @@ presubmits:
         securityContext:
           privileged: true
 
+  - name: pull-kubetest2-gce-legacy-build-up-down
+    decorate: true
+    always_run: true
+    optional: true
+    cluster: k8s-infra-prow-build
+    labels:
+      preset-dind-enabled: "true"
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    annotations:
+      testgrid-dashboards: sig-testing-canaries
+    extra_refs:
+    - base_ref: master
+      org: kubernetes
+      repo: cloud-provider-gcp
+      path_alias: k8s.io/cloud-provider-gcp
+    - org: kubernetes
+      repo: kubernetes
+      base_ref: master
+      path_alias: k8s.io/kubernetes
```

Comment:
this combination does not work without some magic I added to the cloud-provider repo, is this obtained from other place?

Reply:
I have been running this job(the one above it) for a while checking out both repos with the path_alias. It works